nixos/modules/services/web-apps/mediawiki.nix

```
@@ -276,6 +279,13 @@ in
         description = "Contact address for password reset.";
       };
 
+      path = mkOption {
+        type = types.listOf types.package;
+        defaultText = lib.literalExpression "with pkgs; [ diffutils imagemagick ]";
```

Should this not also set the default?
```suggestion
        default = with pkgs; [ diffutils imagemagick ];
        defaultText = lib.literalExpression "with pkgs; [ diffutils imagemagick ]";
```

---

Then we would overwrite imagemagick as soon as we add anything to it. The default is in config and unless you use mkForce, it cannot be overwritten.

---

Am I missing something? I cannot see this in the config changes of this pull request.

---

Ah. missed it.